packages/backend/src/graphql/schema.graphql

```diff
@@ -1,164 +1,185 @@
 type Query {
-  "Fetch an asset"
-  asset(id: String!): Asset
+  "Fetch an asset by its ID."
+  asset(
+    "Unique identifier of the asset."
+    id: String!
+  ): Asset
 
-  "Fetch a page of assets."
+  "Fetch a paginated list of assets."
   assets(
-    "Paginating forwards: the cursor before the the requested page."
+    "Forward pagination: Cursor to start retrieving assets after this point."
```

Comment:
should we specify the cursor is an asset `id`?
(applies to other pagination as well)